sdk/hdinsight/Azure.ResourceManager.HDInsight.Containers/CHANGELOG.md

```diff
@@ -1,4 +1,18 @@
 # Release History
+## 1.0.0-beta.4 (Unreleased)
+
+### Features Added
+
+- Cluster Pool
+    - Enable create cluster pool with user network profile.
+    - Enable get cluster pool available upgrade versions.
+- Cluster
+    - Enable create Ranger cluster.
+    - Enable get cluster available upgrade versions.
+    - Enable set internal ingress.
+    - Enable check if the cluster name is available.
+    - Enable upgrade hot fix for cluster.
+    - Enable upgrade node os for cluster.
```

Comment:
Please add your changelog to the latest unreleased version (aka 1.0.0-beta.3) as a new item